packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -311,19 +322,46 @@ LiveLabel.defaultProps = {
   dir: 'ltr',
 };
 
-const StoryPromo = ({ image, info, mediaIndicator, topStory, ...props }) => (
+const StoryPromo = ({
+  image,
+  info,
+  mediaIndicator,
+  topStory,
+  displayImage,
+  dir,
+  ...props
+}) => (
   <StoryPromoWrapper topStory={topStory} {...props}>
-    <ImageGridItem topStory={topStory}>
-      <ImageContentsWrapper>
-        {image}
-        {mediaIndicator && (
-          <InlineMediaIndicator topStory={topStory}>
+    {displayImage ? (
+      <>
+        <ImageGridItem topStory={topStory}>
+          <ImageContentsWrapper>
+            {image}
+            {mediaIndicator && (
+              <InlineMediaIndicator
+                topStory={topStory}
+                displayImage={displayImage}
+                dir={dir}
+              >
+                {mediaIndicator}
+              </InlineMediaIndicator>
+            )}
+          </ImageContentsWrapper>
+        </ImageGridItem>
+        <TextGridItem topStory={topStory} displayImage={displayImage}>
+          {info}
+        </TextGridItem>
+      </>
+    ) : (
+      <>
+        <TextGridItem topStory={topStory} displayImage={displayImage}>
+          <InlineMediaIndicator displayImage={displayImage} dir={dir}>
```

Comment:
I think we should check if `mediaIndicator` provided before rendering the `InlineMediaIndicator`. 
Something like 
`{mediaIndicator && (<InlineMediaIndicator {...props}>{ children }</ InlineMediaIndicator>)}`